README.md

```diff
@@ -18,23 +18,21 @@ The different modules are used by Terraform workspaces which represent three AWS
 
 These scripts assume that the [Terraform backend has been created in the management account][tf-backend]. This project uses the S3 Terraform backend to store the Terraform state for the different TDR environments.
 
-To start a deployment, run the [TDR Terraform Environments Deploy job in Jenkins][jenkins-job] by clicking 'Build with Parameters' and selecting the environment you want to deploy to. All changes must be deployed first to integration, then staging, then production.
+To start a deployment, run the [TDR Terraform Environments Deploy job in GitHub actions][github-actions-job] by clicking 'Run Workflow' and selecting the environment you want to deploy to. All changes must be deployed first to integration, then staging, then production. 
 
-The deployment will pause when Terraform has determined which changes need to be applied. Review the Terraform plan output by opening the job's console output.
+The deployment will pause when Terraform has determined which changes need to be applied. Review the Terraform plan output by clicking the link provided in the Slack notification. This will be a link to Cloudwatch in the management account so you will need to be logged in to the management AWS account to use this.
```

Comment:
Just a thought, do the junior devs (Thanh and Nigel) have access to the management account logs? If not will need to do something about that.

Reply:
Good point, I'll have a look at the role they're attached to. I can put in a PR for digital services to have a look at if not.